test/cql-pytest/test_materialized_view.py

```diff
@@ -1354,3 +1354,56 @@ def test_create_materialized_view_slash_name(cql, test_keyspace, table1):
     finally:
         # We shouldn't reach here, but if we did, let the test fail cleanly
         cql.execute(f'DROP MATERIALIZED VIEW IF EXISTS {test_keyspace}."/xyz/"')
+
+@pytest.fixture(scope="module")
+def mv1(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, 'p int, c int, x int, primary key (p)') as table:
+        with new_materialized_view(cql, table, '*', 'c, p', 'p is not null and c is not null') as mv:
+            yield mv
+
+# A materialized view may not be written to directly - it can only be written
```

Comment:
I'm not an expert, but `cannot` sounds like it's impossible to do it, while `may not` sounds like it `may or may not happen`.
```suggestion
# A materialized view cannot be written to directly - it can only be written
```